test/python/expected/sql_validation_framework/expected_drop.out

```diff
@@ -0,0 +1,36 @@
+Unexpected drop found for operator class sys.fixeddecimal_ops in file ../../contrib/babelfishpg_common/sql/upgrades/babelfishpg_common--1.0.0--1.1.0.sql
+Unexpected drop found for operator class sys.int2_fixeddecimal_ops in file ../../contrib/babelfishpg_common/sql/upgrades/babelfishpg_common--1.0.0--1.1.0.sql
+Unexpected drop found for operator class sys.int2_fixeddecimal_ops in file ../../contrib/babelfishpg_common/sql/upgrades/babelfishpg_common--1.0.0--1.1.0.sql
+Unexpected drop found for operator class sys.int4_fixeddecimal_ops in file ../../contrib/babelfishpg_common/sql/upgrades/babelfishpg_common--1.0.0--1.1.0.sql
+Unexpected drop found for operator class sys.int4_fixeddecimal_ops in file ../../contrib/babelfishpg_common/sql/upgrades/babelfishpg_common--1.0.0--1.1.0.sql
+Unexpected drop found for operator class sys.int8_fixeddecimal_ops in file ../../contrib/babelfishpg_common/sql/upgrades/babelfishpg_common--1.0.0--1.1.0.sql
+Unexpected drop found for operator class sys.int8_fixeddecimal_ops in file ../../contrib/babelfishpg_common/sql/upgrades/babelfishpg_common--1.0.0--1.1.0.sql
+Unexpected drop found for operator class sys.numeric_fixeddecimal_ops in file ../../contrib/babelfishpg_common/sql/upgrades/babelfishpg_common--1.0.0--1.1.0.sql
+Unexpected drop found for operator class sys.numeric_fixeddecimal_ops in file ../../contrib/babelfishpg_common/sql/upgrades/babelfishpg_common--1.0.0--1.1.0.sql
+Unexpected drop found for operator family sys.fixeddecimal_ops in file ../../contrib/babelfishpg_common/sql/upgrades/babelfishpg_common--1.0.0--1.1.0.sql
+Unexpected drop found for operator family sys.fixeddecimal_ops in file ../../contrib/babelfishpg_common/sql/upgrades/babelfishpg_common--1.0.0--1.1.0.sql
+Unexpected drop found for procedure sys.babelfish_drop_deprecated_function in file ../../contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--1.2.1--2.0.0.sql
+Unexpected drop found for procedure sys.babelfish_drop_deprecated_function in file ../../contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--2.1.0--2.2.0.sql
+Unexpected drop found for procedure sys.babelfish_drop_deprecated_object in file ../../contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--2.0.0--2.1.0.sql
+Unexpected drop found for procedure sys.babelfish_drop_deprecated_view in file ../../contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--2.1.0--2.2.0.sql
+Unexpected drop found for procedure sys.babelfish_remove_object_from_extension in file ../../contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--2.1.0--2.2.0.sql
+Unexpected drop found for procedure sys.create_xp_qv_in_master_dbo in file ../../contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--1.1.0--1.2.0.sql
+Unexpected drop found for procedure sys.sp_babelfish_grant_usage_to_all in file ../../contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--1.1.0--1.2.0.sql
```

Comment:
> CREATE OR REPLACE PROCEDURE sys.sp_babelfish_grant_usage_to_all()
> AS $$
> DECLARE
> 	schema_name text;
> BEGIN
> 	FOR schema_name IN SELECT nspname FROM sys.babelfish_namespace_ext
> 	LOOP
> 		EXECUTE format('GRANT USAGE ON SCHEMA %I TO PUBLIC', schema_name);
> 	END LOOP;
> END;
> $$ LANGUAGE plpgsql;
> 
> CALL sys.sp_babelfish_grant_usage_to_all();
> DROP PROCEDURE sys.sp_babelfish_grant_usage_to_all;

I don't see any unexpected drop for the `PROCEDURE sys.sp_babelfish_grant_usage_to_all` here. Why does it say unexpected drop for other procedures too like `create_xp_qv_in_master_dbo`?

Reply:
The `expected/expected_drop` file contains the drop statements that are expected in the scripts for which the test should pass.
Workflow of framework:
The framework looks for all drop statements in the upgrade scripts and adds them to `output/expected_drop` file. By taking diff of both files, if any drop except those in `expected/expected_drop` is found, the test will fail.

I think it would be better to remove the word unexpected from the file.